src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -36,7 +36,7 @@ public static void TestFixtureSetup(TestContext context)
             OverrideEntityContainer("Planet", _containerName);
             OverrideEntityContainer("Earth", _containerName);
             OverrideEntityContainer("StarAlias", _containerName);
-            OverrideEntityContainer("TagAlias", _containerName);
```

Comment:
Is `TagAlias` no longer used? If yes, did we remove from all the occurrences?

Reply:
Yes, it's used here. https://github.com/Azure/data-api-builder/blob/a52db81da1d9a2248f1cf83b978a58dae5290dda/src/Service.Tests/CosmosTests/QueryFilterTests.cs#L665